src/dotnet/ToolPackage/CommandSettingsListId.cs

```diff
@@ -39,7 +42,11 @@ public bool Equals(CommandSettingsListId other)
                    string.Equals(
                        RuntimeIdentifier,
                        other.RuntimeIdentifier,
-                       StringComparison.OrdinalIgnoreCase);
+                       StringComparison.OrdinalIgnoreCase) &&
+                   string.Equals(
+                       CommandName,
+                       other.CommandName,
+                       StringComparison.InvariantCulture);
```

Comment:
Does this mean that command names will be case sensitive even on Windows?

Reply:
Looks like in the HashCode function you're using OrdinalIgnoreCase, so there seems to be a mismatch here.

Reply:
Good catch, for global tools it will fail/not fail to create the shim. I think it's better to use a same strategy. Since the same manifest could be shared between Windows and Non Windows, and one user cannot restore and one can. I think it should all be case insensitive? To be more strict? (more likely to block and less surprise?).

And yes, I should keep Hashcode in sync.

Reply:
Discussed offline, we should block on manifest with commands with same name with casing different. And resolve case insensitive on both platform for now.

Changed to OrdinalIgnoreCase and aligned it with hashcode.